debian/acrn-board-inspector/acrn-board-inspector

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
It definitely would make sense to put that into board_inspector itself. But since a stumbled upon it several times I had the feeling I had to put it somewhere. And since I need a wrapper anyway to wrap the python call I have put it in here.